scripts/aggregate-translations/aggregate-translations.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path');
+const fse = require('fs-extra');
+const glob = require('glob');
+const supportedLocales = require('./i18nSupportedLocales');
+
+const aggregateMessages = require('./aggregate-messages');
+const writeAggregatedTranslations = require('./write-aggregated-translations');
+const writeI18nLoaders = require('./write-i18n-loaders');
+
+const defaultSearchPatterns = baseDirectory => ([
+  path.resolve(baseDirectory, 'translations'), // root level translations
+  path.resolve(baseDirectory, 'node_modules', 'terra-*', 'translations'), // root level dependency translations
+  path.resolve(baseDirectory, 'packages', 'terra-*', 'translations'), // package level translations
+  path.resolve(baseDirectory, 'packages', 'terra-*', 'node_modules', 'terra-*', 'translations'), // package level dependency translations
+]);
+
+const customDirectories = (baseDirectory, directories) =>
+  (directories.map(dir => path.resolve(baseDirectory, dir)));
+
+const aggregatedTranslations = (options) => {
+  const baseDir = (options || {}).baseDir || process.cwd();
```

Comment:
would it be better to do a store options in a temp variable at the beginning of the method instead of doing options || {} for all these variables?

Reply:
This options stuff will be updated with the forthcoming webpack4 update. :)